lib/context2d.js

```diff
@@ -10,264 +10,10 @@
  * Module dependencies.
  */
 
-const bindings = require('./bindings')
-const parseFont = require('./parse-font')
-const Context2d = module.exports = bindings.CanvasRenderingContext2d
-const CanvasGradient = bindings.CanvasGradient
-const CanvasPattern = bindings.CanvasPattern
-const ImageData = bindings.ImageData
-const DOMMatrix = require('./DOMMatrix').DOMMatrix
+const bindings = require('./bindings');
+const parseFont = require('./parse-font');
+const {DOMMatrix} = require('./DOMMatrix');
```

Comment:
It would be nice to follow [standard](https://github.com/feross/standard) on all new code that we add ☺️ 

```suggestion
const { DOMMatrix } = require('./DOMMatrix')
```

Since this entire file is rewritten you could add it to [the list here](https://github.com/Automattic/node-canvas/blob/cc6c173bfd1d77ef1fd3552d655c0a4545218a2a/package.json#L28) as well 🎉 

(btw. I know this is a bit of a complicated way to convert a codebase 😄 but it's the one being used by Express.js as well. The goal of it is to not destroy the diffs and ability to backport stuff by rewriting all files at once, rather we just fix the lines that are already touched by a diff ☺️)